docs/en/getting-started/install.md

```diff
@@ -90,6 +90,16 @@ sudo clickhouse-client-$LATEST_VERSION/install/doinst.sh
 
 For production environments, it’s recommended to use the latest `stable`-version. You can find its number on GitHub page https://github.com/ClickHouse/ClickHouse/tags with postfix `-stable`.
 
+### From precompiled binaries for non-Linux OS {#from-binaries-non-linux}
+
+For non-Linux operating systems, ClickHouse builds are provided as a cross-compiled binary from the latest commit of the master branch.
+
+- macOS — `curl -O 'https://clickhouse-builds.s3.yandex.net/master/macos/clickhouse'`
+- AArch64 — `curl -O 'https://clickhouse-builds.s3.yandex.net/master/aarch64/clickhouse'`
+- FreeBSD — `curl -O 'https://clickhouse-builds.s3.yandex.net/master/freebsd/clickhouse'`
+
+These builds are not recommended for use in production environment. Not all ClickHouse features are present in them.
```

Comment:
```suggestion
These builds are not recommended for use in production environments because they are less thoroughly tested, but you can do so on your own risk. They also have only a subset of ClickHouse features available.
```